gui-wm/hyprland/hyprland-0.30.0.ebuild

```diff
@@ -77,6 +77,11 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES=(
+	"${FILESDIR}/hyprland-0.30.0-no-wlroots-automagic-r1.patch"
+	"${FILESDIR}/hyprland-0.30.0-3400-fix-build.patch"
+)
+
 pkg_setup() {
```

Comment:
Please update the check in pkg_setup to need GCC 13 because of std::format.

Reply:
Done, thanks !